### DISCUSSION TITLE
Welcome to docs Discussions!

### DISCUSSION THREAD

**Discussion by velddev** [top-gg/docs#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SubhadipGitHub** [comment]
Hi Team,

There is an issue with docs related to api where the url for finding one bot info is given as -
https://top.gg/api/bot/XXXXXX (Incorrect)
https://top.gg/api/bots/XXXXXX (Correct)

Hope this will be corrected soon for future reference.Cheers!
![image](https://user-images.githubusercontent.com/29476358/108026770-02678c80-704f-11eb-92d2-8029c1df5ac2.png)
